app/commands/schedule.rb

```diff
@@ -6,17 +6,59 @@ class Schedule < Base
     # 🚩 *The Future of library dependency management of Ruby*
 
     def call(message)
-      schedule_keyboard = Telegram::Bot::Types::InlineKeyboardMarkup.new(
-        inline_keyboard: [[
-          Telegram::Bot::Types::InlineKeyboardButton.new(text: 'June 1, Sat', callback_data: { command: 'schedule', args: 1 }.to_json),
-          Telegram::Bot::Types::InlineKeyboardButton.new(text: 'June 2, Sun', callback_data: { command: 'schedule', args: 2 }.to_json)
-        ]]
-      )
+      case message
```

Comment:
this code could also be moved to `Commands::Base` later if we accept the concept that each command handles both messages and their callbacks